explore/src/main/scala/explore/itc/ItcGraphQueryer.scala

```diff
@@ -98,10 +98,16 @@ case class ItcProps(
 
   val targets: List[ItcTarget] = itcTargets.foldMap(_.toList)
 
-  private val queryProps: List[Option[?]] =
-    List(itcTargets, finalConfig, wavelength, instrumentConfig, signalToNoise)
+  private val queryProps =
+    (wavelength,
+     signalToNoise,
+     signalToNoiseAt,
+     constraints.some,
```

Comment:
I don't think `constraints` is needed. All that we need from it is already factored into `instrumentConfig`.